src/components/Header/CKBNodeComp/CKBNodeModal.tsx

```diff
@@ -0,0 +1,44 @@
+import { useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useCKBNode } from '../../../hooks/useCKBNode'
+import CommonModal from '../../CommonModal'
+import { HelpTip } from '../../HelpTip'
+import { Switch } from '../../ui/Switch'
+import CloseIcon from '../../../assets/modal_close.png'
+import styles from './style.module.scss'
+
+export const CKBNodeModal = ({ onClose }: { onClose: () => void }) => {
+  const { t } = useTranslation()
+  const ref = useRef<HTMLDivElement>(null)
+  const { activate, setActivate } = useCKBNode()
+
+  return (
+    <CommonModal isOpen onClose={onClose}>
+      <div ref={ref} className={styles.modalWrapper}>
+        <div className={styles.contentWrapper}>
+          <div className={styles.modalTitle}>
+            <p>{t('navbar.node')}</p>
+            <button type="button" onClick={onClose} className={styles.closeBtn}>
+              <img src={CloseIcon} alt="close icon" />
+            </button>
+          </div>
+
+          <div className={styles.switcher}>
+            <label htmlFor="node-connect-mode">{t('navbar.node_connect_mode')}</label>
+            <HelpTip title={t('navbar.node_connect_tooltip')} />
+            <Switch
+              id="node-connect-mode"
+              style={{ marginLeft: 'auto' }}
+              checked={activate}
+              onCheckedChange={checked => setActivate(checked)}
+            />
+          </div>
+
+          <button type="button" className={styles.doneBtn} onClick={onClose}>
+            {t('nervos_dao.done')}
```

Comment:
i18n `done` should be defined specifically to avoid reusing it from an irrelavant module.